### ISSUE TITLE
Screenreader doesn't read clearance level paws in tables

### ISSUE THREAD

**Issue by RomijnHumanoids** [SSC-ICT-Innovatie/nl-kat-coordination#4205]
**Describe the bug**
While performing accessibility check for ticket #3812, I came across a bug where screenreader reads table cells with a clearance level as blank. First discovered on feat toggle branch, but also validated on main branch. It should of course tell what clearance level is visualised by the coloured paw icons. Maybe we can implement a solution where not only the paws component is coded/labeled with aria labels that make screenreader read it, but also defined as such on hover (for users that might not get the visual clues).

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'object page'
2. Click on 'screenreader keys (on Mac: command F5)'
3. Scroll down to 'a table cell with a clearance level in it. (on Mac: control option right arrow'
4. See/hear 'blank'

**Expected behavior**
Repeated: _First discovered on feat toggle branch, but also validated on main branch. It should of course tell what clearance level is visualised by the coloured paw icons. Maybe we can implement a solution where not only the paws component is coded/labeled with aria labels that make screenreader read it, but also defined as such on hover (for users that might not get the visual clues)._

**Screenshots**

Report flow, step 1 'object selection': 
<img width="1753" alt="Image" src="https://github.com/user-attachments/assets/2e40f29b-b7ba-4636-9dff-5782071cc973" />

Objects page:
<img width="1431" alt="Image" src="https://github.com/user-attachments/assets/21c43f3a-d5de-46a0-923e-e02102fbcf5b" />

**OpenKAT version**
Note the release tag (and if possible: the installation method) here.
If it concerns an in-development version, note the branch(es) and commit hash(es) here as well.

**Desktop (please complete the following information if relevant):**

- OS: [MacOS 15.3.2]
- Browser: [Chrome]
- Version: [Current version on main branch 24 march 2025]

**Additional context**
Other accessibility remarks about the stepper component and toggle feature component, that could possibly be solved in the same ticket,  :
- The stepper component (in the report flow) is, just like the new toggle component, indicated as a 'list' and the steps inside it are then indicated as a 'step' with a 'link' inside. Users with a screen reader are expected to understand that it is a stepper/progress bar by hearing 'current step' after hearing 'list'. It would be more optimal if the component is indicated as a 'stepper' right away, and the step is read out in one go instead of first the step and then the link inside it. (see screenshots).
<img width="1710" alt="Image" src="https://github.com/user-attachments/assets/d4cb94e6-9fb8-45f7-9f61-7ad4d616f88a" />

<img width="1724" alt="Image" src="https://github.com/user-attachments/assets/8a3d7591-e180-4e1e-934b-bea97ab715eb" />

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/2806f5f6-1c3d-4cc3-9920-9143e5d079af" />

- The toggle feature component has similar issues of which remarks are noted in the accessibility check on ticket #3812.
